app.config.js

```diff
@@ -10,10 +10,10 @@ module.exports = {
 
   // List of chainIds which metadata cache queries will return by default.
   // This preselects the Chains user preferences.
-  chainIds: [3, 4],
+  chainIds: [2021000],
```

Comment:
question, suggestion(non-blocking): Can we name this magic number and create an additional variable like:

```
GaiaXChainId: 202100
chaindIds: [GaiaXChainId]
```

Reply:
Should I create the ew variable in a separate file (i.e. inside src/models) or directly inside here?

Reply:
Good question. I believe we might also use the same variable somewhere else so a place for constants might be something to think about. Let's keep it in the file for now. We will discuss a constant file in our next daily meeting.

Reply:
I propose using the chains.config.ts file
This is our method for overwriting network specific configurations

have a look at this proposal:
we can import variables to app-config like this: `chains.GaiaXTestnet.networkId`
```
// chain configs in ocean.js ConfigHelperConfig format
// see: https://github.com/oceanprotocol/ocean.js/blob/e07a7cb6ecea12b39ed96f994b4abe37806799a1/src/utils/ConfigHelper.ts#L8
// networkId is required, since its used to look for overwrites
// all other fields are first loaded from ocean.js and are optional
import { ConfigHelperConfig } from '@oceanprotocol/lib'
import { metadataCacheUri } from './app.config'

type ConfigHelperConfigOverwrite = Partial<ConfigHelperConfig> &
  Required<Pick<ConfigHelperConfig, 'networkId'>>

const chains: { [name: string]: ConfigHelperConfigOverwrite } = {
  GaiaXTestnet: {
    networkId: 2021000,
    nodeUri: 'https://rpc.gaiaxtestnet.oceanprotocol.com/',
    metadataCacheUri: metadataCacheUri,
    providerUri: 'https://provider.gaiax.delta-dao.com/',
    explorerUri: 'https://blockscout.gaiaxtestnet.oceanprotocol.com/'
  },
  CatenaXTestnet: {
    networkId: 2021001,
    nodeUri: 'https://rpc.catenaxtestnet.oceanprotocol.com/',
    metadataCacheUri: metadataCacheUri,
    providerUri: 'https://provider.catenax.delta-dao.com/',
    explorerUri: 'https://blockscout.catenaxtestnet.oceanprotocol.com/'
  },
  RinkebyTestnet: {
    networkId: 4,
    providerUri: 'https://provider.rinkeby.delta-dao.com'
  },
  RopstenTestnet: {
    networkId: 3,
    providerUri: 'https://provider.ropsten.delta-dao.com'
  }
}

export default chains
```